.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Description of work.
+
+**To test:** Instructions for testing.
+
+Fixes #xxxx.
+
+Either link to [Release notes](http://www.mantidproject.org/Release_Notes_3.7) change-set or state "Does not need to be in the release notes."
```

Comment:
This is how CMake handle their release notes: https://github.com/Kitware/CMake/tree/master/Help/release. That might be too simplistic for our needs though (and off topic for this conversation)